charts/tracing-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tracing-apps
 description: Argo CD app-of-apps config for tracing applications
 type: application
-version: 0.33.0
+version: 1.0.0
```

Comment:
```suggestion
version: 0.34.0
```

Reply:
too late, it's 1.0.0 now and there is no official way back

this isn't a huge issue, details on 0.x are in the upstrteam semver docs